### PR TITLE
Be/email server

### PR DESCRIPTION
**Pull request by thawk55**
endpoint to send a notification as an email

### CODE REVIEW

File: api/email/email.controller.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    text: notification.message
+  };
+
+  request.post({url: url, formData: formData}, function(error, resp, body){
+    if(error)
+      res.send(error);
+    else{
+      res.send({message:"message queued to send."});
+    }
+  }).auth('api', 'key-2052f3bd87a939ab189aacf1508753b6');
```
Comment (jeffreykbacon) [inline api/email/email.controller.js]: We should be able to put the key in an environment variable.

File: api/swagger/swagger.json [hunk 2]
```diff
@@ -1328,6 +1360,14 @@
                     "type": "string"
                 }
             }
+        },
+        "Email": {
```
Comment (jeffreykbacon) [inline api/swagger/swagger.json]: Is this correct?

**Comment by jeffreykbacon** [comment]
Just a couple suggestions, otherwise :+1:

File: api/swagger/swagger.json (hunk 2 above)
Comment (thawk55) [inline api/swagger/swagger.json]: No, it's uppercase just like the other models.

File: api/email/email.controller.js (hunk 1 above)
Comment (thawk55) [inline api/email/email.controller.js]: I plan to once it gets merged in :)

**Comment by tdecker91** [comment]
Looks good. Merge when ready.

**Pull request merged**